en/docs/get-started/development-kickstart.md

```diff
@@ -588,7 +588,7 @@ Optionally, you can install and start the Integration Control Plane (ICP) to obs
 
         <a href="{{base_path}}/assets/img/develop/mi-for-vscode/qsg/icp-login.png"><img src="{{base_path}}/assets/img/develop/mi-for-vscode/qsg/icp-login.png" alt="ICP login" width="80%"></a>
 
-    2. Use the following sign in details.
+    2. Use the following sign-in details.
```

Comment:
Shall we keep this as 'sign in' since it aligns with the Microsoft writing style guide? https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/sign-in-sign-out

Reply:
Reverted with https://github.com/wso2/docs-mi/pull/797/commits/766cdc6eff6eb1b13d78a7865a96942220f75cda